aider.el

```diff
@@ -381,6 +402,78 @@ The command will be formatted as \"/ask \" followed by the text from the selecte
   :lighter " Aider"
   :keymap aider-minor-mode-map)
 
+(defvar aider-prog-mode-map (make-sparse-keymap)
+  "Keymap for Aider commands in programming modes.")
+
+;;;###autoload
+(define-minor-mode aider-prog-mode
+  "Minor mode for Aider programming mode integration."
+  :lighter " Aider"
+  :keymap aider-prog-mode-map)
+
+;;;###autoload
+(defun aider-setup-doom-bindings ()
```

Comment:
I personally feel it would be better to let the key-binding to global, and do not use aider-prog-mode. So the pattern for DOOM emacs is in sync with the Vanilla emacs